code/modules/reagents/reagents.dm

```diff
@@ -49,9 +49,52 @@
 	var/heating_sound = 'sound/effects/bubbles.ogg'
 	var/list/heating_products
 
+	var/constantMetabolism = FALSE	// if metabolism factor should not change with volume or blood circulation
+
+	var/NSA = 10 // Nerve system accumulations
```

Comment:
All caps var names look like defines, it would probably be better if this was lowercased, or snake cased somehow.